root/components/common-macros.tt

```diff
@@ -887,11 +887,14 @@ note-[%- edit.id -%]-[%- loop.count() -%]
 [%- END -%]
 
 [% MACRO votename(note) BLOCK %]
-  [% edit = note.edit %]
+  [%- is_owner = edit.editor_id == note.editor_id -%]
+  [%- edit = note.edit -%]
```

Comment:
Shouldn't `edit` be declared first since it's referenced in `is_owner = edit.editor_id == note.editor_id`?

Reply:
It seemed to work fine, but I'm not quite sure how now that you mention this. Maybe the equality was only checked when is_owner was used? I'm confused. Just TT things I guess. Anyway, will make the change.